paimon-core/src/main/java/org/apache/paimon/table/system/ManifestsTable.java

```diff
@@ -212,12 +214,16 @@ private static List<ManifestFileMeta> allManifests(FileStoreTable dataTable) {
                                 snapshotId, earliestSnapshotId, latestSnapshotId));
```

Comment:
String.format(
                                "Specified parameter %s = %s is not exist, you can set it in range from %s to %s.",
                                SCAN_SNAPSHOT_ID.key(), snapshotId, earliestSnapshotId, latestSnapshotId));